docs/source/user_guide/best_practices.rst

```diff
@@ -170,3 +170,79 @@ this operation should be avoided.
 
 See `Operations on different DataFrames <options.rst#operations-on-different-dataframes>`_ for more details.
 
+
+Use Koalas APIs directly whenever possible
+------------------------------------------
+
+Although Koalas has most of the pandas-equivalent APIs with pandas, there are several APIs not implemented yet or explicitly unsupported.
```

Comment:
nit: `... most of the pandas-equivalent APIs, there are ...` or `... most of the equivalent APIs to pandas, there are ...`?